### DISCUSSION TITLE
postgres 安装失败

### DISCUSSION THREAD

**Discussion by MystiPanda** [1Panel-dev/1Panel#5922] [Q&A]
### 1Panel 版本

v1.10.13-lts

### 问题描述

升级最新版本之后安装postgresql，全默认设置点击安装后会跳转到首页，没有任何报错信息。

（记得上个版本安装过postgresql，是可以正常安装的）

### 重现步骤

使用v1.10.13-lts版本1Panel
在应用商店点击安装postgresql

### 附加信息

没有看到有相关的日志，如果需要烦请告知在哪里获取相关日志

**Comment by wanghe-fit2cloud** [comment]
浏览器 `IP:Port ` 方式访问，存在上述问题吗？
> 本地环境未复现。

**Reply by MystiPanda** [reply]
感谢回复，确实是通过域名访问的，经您提醒之后我看了下浏览器的网络请求，install请求因为携带恶意参数被1Panel拦截了，确实通过ip访问就正常了

**Reply by AlphaZ19** [reply]
+1 我也出现了这个问题，然后通过 IP:Port  方式访问并成功安装
